contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -255,7 +287,11 @@ func (h *htlcIncomingContestResolver) Resolve() (ContractResolver, error) {
 			return &h.htlcSuccessResolver, nil
 
 		case hodlItem := <-hodlChan:
-			htlcResolution := hodlItem.(invoices.HtlcResolution)
+			htlcResolution, ok := hodlItem.(invoices.HtlcResolution)
```

Comment:
can make `hodlChan` type ` chan invoices.HtlcResolution` now?

Reply:
The original reason for having it typed `interface{}` was because it was used with a queue.